src/collective/glossary/browser/views.py

```diff
@@ -36,6 +39,15 @@ class GlossaryView(BrowserView):
 
     """Default view of Glossary type"""
 
+    def language(self):
```

Comment:
this can be done using the [API](https://docs.plone.org/develop/plone.api/docs/portal.html#portal-get-current-language-example):

```python
language = api.portal.get_current_language()
```

Reply:
Changed.